src/robot_interface/telemetry/payloads.py

```diff
@@ -16,3 +21,31 @@ class TelemetryBatteryPayload:
     battery_level: float
     robot_id: str
     timestamp: datetime
+
+
+@dataclass
+class VideoStream:
+    name: str
+    url: str
+
+
+@dataclass
+class RobotStatusPayload:
```

Comment:
Should we not include current_isar_state, current_mission_id, current_task_id and current_step_id until we need it for something? That way we can add it as needed and only supply the information that Flotilla actually use

Reply:
I would argue this message is "general" in the sense that it's the status of the ISAR instance. It is not a message that should exist for Flotilla because they are modular applications. It's Flotilla's choice to use this information for what it needs it to.

If other applications should use ISAR this information is the status. Also, it might be good to have it for the timeseries logging. Unless we go with seperate publishers for each one.

Reply:
Agreed